### DISCUSSION TITLE
Giscus Comments Test · Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#10] [General]
# Giscus Comments Test · Developer Portal

The developer resources in just one place!

http://localhost:1313/blog/giscus_test_post/

**Comment by pedrominatel** [comment]
It's working!

**Comment by zikalino** [comment]
Ita, gisqus est optimus in mundo!

**Comment by f-hollow** [comment]
Logged in directly now without any additional routing. Hehe :)

**Comment by marekmatej** [comment]
Working :)

**Comment by ustccw** [comment]
Oops, where have I been teleported? Anyway, I have to leave a comment first.

**Comment by lucasssvaz** [comment]
Nice
